src/pages/on-chain-governance/create/index.page.tsx

```diff
@@ -147,7 +149,12 @@ export default function ProposalDetailPage() {
   return (
     <>
       <Head title="Create Proposal" />
-      <Container className="mt-10 md:mt-12 px-6 lg:px-[312px]">
+      <Container
+        className={classNames(
+          "mt-10 md:mt-12 px-6",
+          windowWidth < 1280 ? "lg:px-6" : "lg:px-[312px]"
```

Comment:
we can use `xl` breakpoint for this
ref: https://tailwindcss.com/docs/responsive-design